src/Animals/BigDog.ts

```diff
@@ -0,0 +1,12 @@
+import { AnimalNames } from '../Enums/AnimalNamesEnum';
+import { ProtectHerdInterface } from '../Interfaces/ProtectHerdInterface';
+import { Animal } from './Animal';
+
+export class BigDog extends Animal implements ProtectHerdInterface {
+  constructor() {
+    super(AnimalNames.BIG_DOG, 36);
```

Comment:
What are these numbers? 36, 72 ?

Reply:
These numbers are value of Animal in lowest currency

Reply:
Animal Dollars :)